forem/forem
# Show tips on the post help content + update styles

## What type of PR is this? (check all applicable)


## Description

## Related Tickets & Documents

- Closes #20414

## QA Instructions, Screenshots, Recordings

1. Go to `/new`
2. Move down the inputs on the page to see the updates to new contextual help on the right hand of the page
3. Move mouse to bottom of the page near the publish button to see new tips location.

### UI accessibility checklist
_If your PR includes UI changes, please utilize this checklist:_

## Added/updated tests?
_We encourage you to keep the code coverage percentage at 80% and above._
